Paraswap-Agents/AdminOperations/README.md

```diff
@@ -0,0 +1,63 @@
+# Admin Operations Agent
```

Comment:
It would be nice if you had some examples transactions for improved testing (the transaction details for on-chain transactions are way more detailed than the ones in our test harnesses)
If it's not feasible to find transactions then it's fine, but I'd definitely say it's a nice-to-have.

Reply:
I definitely agree. I added the test transaction that I found and generates a finding.